CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# Contributing to Tech Phantoms
+
+A big welcome and thank you for considering contributing to [Tech Phantoms](techphantoms.co) open source project! It’s people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing thIS open source project. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+## Quicklinks
+
+* [Code of Conduct](#code-of-conduct)
+* [Issues](#issues)
+* [How Can I Contribute?](#how-can-i-contribute)
+    * [Reporting Bugs](#reporting-bugs)
+    * [Suggesting Enhancements](#suggesting-enhancements)
+    * [Code Contribution](#code-contribution)
+* [Pull Requests](#pull-requests)
+* [Getting Help](#getting-help)
+
+<br/>
+
+## Code of Conduct
+
+We take our open source community seriously and hold ourselves and other contributors to high standards of communication. By participating and contributing to this project, you agree to uphold our [Code of Conduct](https://github.com/Tech-Phantoms/Tech-Phantoms/blob/canary/CODE_OF_CONDUCT.md).
+
+<br/>
+
+## Issues
+
+Issues should be used to report problems with the website, request a new feature, or to discuss potential changes before a PR is created. When you create a new Issue, a template will be loaded that will guide you through collecting and providing the information we need to investigate.
+
+If you find an Issue that addresses the problem you're having, please add your own reproduction information to the existing issue rather than creating a new one. Adding a [reaction](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) can also help be indicating to our maintainers that a particular problem is affecting more than just the reporter.
```

Comment:
```suggestion
If you find an Issue that addresses the problem you're having, please add your reproduction information to the existing issue rather than creating a new one. Adding a [reaction](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) can also help be indicating to our maintainers that a particular problem is affecting more than just the reporter.
```